vdgsa_backend/accounts/templates/user_account/membership_renewal.tmpl

```diff
@@ -59,24 +60,28 @@
   method="post"
   action="{% url 'purchase-subscription' pk=user.pk %}"
 >
-  {% include 'utils/form_body.tmpl' with form=form %}
 
-  <div class="redirect-info-msg my-2">
-    Clicking "Checkout" will redirect you to Stripe. <br>
-    Please enter your payment info there.
-  </div>
-  <button type="submit" class="btn btn-primary">Checkout</button>
-  {% if user.subscription is not None %}
-  <button
-    id="hide-purchase-subscription"
-    type="button"
-    class="btn cancel-button"
-    data-bs-toggle="collapse"
-    data-bs-target="#purchase-subscription-form"
-    aria-controls="purchase-subscription-form"
-  >
-    Cancel
-  </button>
+  {% if can_renew_membership and user.subscription.membership_type != 'lifetime' %}
```

Comment:
I think there's logic that hides the "renew membership" link if it's a lifetime membership. Suggest removing `and user.subscription.membership_type != 'lifetime'`